spk/tt-rss/src/conf/resource

```diff
@@ -0,0 +1,26 @@
+{
+    "php56": {
```

Comment:
The point with all those workers is to leave the current PHP installations the task of picking up the app specific configuration files and install them in their appropriate location